demo-plugin/http-server.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -u
+
+PUBLIC_PATH="$1"
+CERT_PATH="$2"
+KEY_PATH="$3"
+shift
+SERVER_OPTS="$@"
+
+http-server $PUBLIC_PATH -p 9001 -S true -C $CERT_PATH -K $KEY_PATH -c-1 --cors $SERVER_OPTS
```

Comment:
we can do something like this: https://github.com/red-hat-storage/odf-console/blob/391c37fa0bd4c1bad7d7c9733a3d308c90ec4728/Dockerfile#L15
`-S true` will signify https is always enabled, even when certs are not passed down to this script...

Reply:
ack